exporter/azure_monitor/common/version.go

```diff
@@ -0,0 +1,4 @@
+package common
+
+var Opencensus_version = "0.3.dev0"
+var Ext_version = "0.3.dev0"

```

Comment:
Is this format following Golang convention?

Reply:
I might just go ahead and delete these variables. There seems to be better ways to get these version numbers. I'm actually not sure what the extenstion version is for. Is this for golang?

Reply:
The information we need here:
1. what's the language runtime version.
2. what's the opencensus SDK version.
3. what's the exporter version (which we call "extension version" in the Python exporter).